app/controllers/api/v1/lobbies_controller.rb

```diff
@@ -1,27 +1,24 @@
-module Api::V1
-  class LobbiesController < ApplicationController
+class Api::V1::LobbiesController < Api::V1::ApplicationController
 
-    before_action :check_user_membership, only: [ :show ]
+  before_action :check_user_membership, only: [ :show ]
 
-    def create
-      new_lobby = Lobby.create
-      new_lobby.add_user(current_user)
-      render json: new_lobby, serializer: LobbySerializer
-    end
-
-    def show
-      render json: lobby, serializer: LobbySerializer
-    end
+  def create
+    new_lobby = Lobbies::Creator.new.create_new_lobby(current_user)
```

Comment:
`Lobbies::Creator.by_user(current_user)` or similar.  This API is a little strange.